shared/css/phase1-design-system.scss

```diff
@@ -0,0 +1,392 @@
+@import 'color';
+@import 'barlowSemiCondensed-font';
```

Comment:
I don't think we need this anymore since these variables are in the `font.scss` file now.

Reply:
we need this cause it imports the font itself :(

Reply:
Ah ok, that makes sense!